staticx/api.py

```diff
@@ -194,6 +203,16 @@ def add_library(self, libpath, exist_ok=False):
         self._added_libs[arcname] = linklib
 
 
+    def audit_library(self, path):
+        """Audit a shared library to check for problems
+
+        Specifically:
+        - See if it uses the problematic DT_RUNPATH (seee #172)
+        """
+        # TODO: Is RPATH a potential problem here, too?
```

Comment:
As indicated by this TODO, I think `RPATH` is potentially problematic, too.

While `RUNPATH` *prevents* the top-level `RPATH` from being used, causing *library not found* errors, a library with `RPATH` could cause a library on the target system to be preferred.

I think we also need to handle `RPATH`. Note that #75 "added support" for `RPATH`, but was only concerned about `$ORIGIN`-relative paths, which probably wouldn't cause a problem in a staticx executable. However, absolute and `..` paths could cause an inadvertent target-system-library load. We could check to see if `RPATH` is safe, but there's no point since Staticx's job is to bundle everything up.